lib/ruport/data/record.rb

```diff
@@ -55,10 +55,10 @@ def initialize(data,options={})
       case(data)
       when Array
         @attributes = options[:attributes] || (0...data.length).to_a
-        @data = @attributes.inject({}) { |h,a| h.merge(a => data.shift) }
+        @data = @attributes.each_with_object({}) { |a, h| h[a.freeze] = data.shift }
```

Comment:
This is essentially:
`Hash[@attributes.zip(data)]`

@jrafanie did you benchmark that?

(this is probably too late to do that work though)

Reply:
yea, changed my mind here

```suggestion
        @data = Hash[@attributes.map(&:freeze).zip(data)]
```